tests/Metadata/MetaDataBaseTest.php

```diff
@@ -312,6 +312,7 @@ public function testInvalidField(string $expectedField, string $expectedType): v
                 $newValue = 'Abb';
                 break;
             case 'array':
+            case 'iterable':
```

Comment:
This is needed because `AllValidator` looks for iterables, which includes arrays, but when it errors, it says "must be an iterable".